apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisHeader.vue

```diff
@@ -104,7 +103,7 @@ const handleDeleteQueryConfirm = async () => {
                     </div>
                     <div class="title-right-extra currency-wrapper">
                         <span class="label">{{ $t('BILLING.COST_MANAGEMENT.COST_ANALYSIS.CURRENCY') }}:</span>
-                        <span>{{ state.currencySymbol }}{{ state.currency }}</span>
+                        <span>{{ CURRENCY_SYMBOL.USD }}{{ CURRENCY.USD }}</span>
```

Comment:
Issue: This `currency` information should be based on the currency that the dataSource has.